tbx/project_styleguide/templates/patterns/organisms/header/header.html

```diff
@@ -1,43 +1,65 @@
 {% load wagtailcore_tags wagtail_cache navigation_tags %}
 {% wagtail_site as current_site %}
-<header class="header grid">
-    {% include "patterns/atoms/skip-link/skip-link.html" %}
-    <a href="/" aria-label="Homepage link" class="grid__header-logo header__logo-link">
-        {% include "patterns/atoms/icons/icon.html" with name="logo" classname="logo" %}
-    </a>
-    <div class="grid__header-nav">
-        {# Desktop navigation #}
-        <nav aria-label="Main navigation" class="primary-nav-desktop" data-desktop-menu>
-            {% wagtailcache 600 "primarynav" current_site.pk is_pattern_library %}
-                <ul data-primary-nav class="primary-nav-desktop__list">
-                    {% primarynav %}
-                </ul>
-            {% endwagtailcache %}
-        </nav>
+<header class="header">
+    <div class="grid">
+        {% include "patterns/atoms/skip-link/skip-link.html" %}
+        <div class="grid__header-nav">
+            {# Primary desktop navigation #}
+            <nav aria-label="Main navigation" class="primary-nav-desktop" data-desktop-menu>
+                {% wagtailcache 600 "primarynav" current_site.pk is_pattern_library %}
+                    <ul class="primary-nav-desktop__list">
+                        <li class="primary-nav-desktop__item">
+                            <a href="/" aria-label="Homepage link" class="primary-nav-desktop__link">
+                                <span class="primary-nav-desktop__text">Home</span>
+                            </a>
+                        </li>
+                        {% primarynav %}
+                    </ul>
+                {% endwagtailcache %}
+            </nav>
 
-        {# Mode switcher desktop #}
-        {% include "patterns/molecules/mode_switcher/mode_switcher.html" %}
-    </div>
+            {# Primary mobile menu toggle #}
+            <button class="header__primary-menu-toggle" data-primary-mobile-menu-toggle aria-haspopup="true" aria-expanded="false" aria-label="Mobile menu toggle">
+                Home
+                {% include "patterns/atoms/icons/icon.html" with name="chevron" classname="header__primary-menu-toggle-icon" %}
+            </button>
 
-    {# Mobile menu toggle #}
-    <div class="header__menu-toggle grid__header-toggle">
-        <span class="sr-only">Menu</span>
-        <button class="button-menu-toggle" data-mobile-menu-toggle aria-haspopup="true" aria-expanded="false" aria-label="Mobile menu toggle">
-            <span class="button-menu-toggle__line"></span>
-            <span class="button-menu-toggle__line"></span>
-            <span class="button-menu-toggle__line"></span>
-            <span class="button-menu-toggle__line"></span>
-        </button>
-    </div>
+            {# Mode switcher desktop - hide for now #}
+            {% include "patterns/molecules/mode_switcher/mode_switcher.html" %}
```

Comment:
Suggestion: when this gets added in properly it might be worth putting it after `<nav aria-label="Main navigation"  class="primary-nav-mobile" data-primary-mobile-menu>` so that when you open the mobile menu with a keyboard the first thing you'll tab to is the menu, rather than the mode switcher : )

Reply:
Although i've just seen that you're doing more work on the mobile menu toggle button in the future so you can ignore this if that's the case!

Reply:
It's actually a valid point - I've made a relevant change (and another related one - tweaked paddings so that focus styles look better on the first and last menu item)